store/zookeeper/zookeeper.go

```diff
@@ -13,7 +13,9 @@ const (
 	// SOH control character
 	SOH = "\x01"
 
-	defaultTimeout = 10 * time.Second
+	defaultTimeout = 10 * time.Second	
+
+	syncRetryLimit = 5
```

Comment:
Any reason to choose 5? I don't know but it seems high. How much delay this may introduce?